services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/client/DataPlaneClient.java

```diff
@@ -17,14 +17,23 @@
 
 import com.futurewei.alcor.dataplane.entity.HostGoalState;
 import com.futurewei.alcor.schema.Goalstate.GoalState;
+import com.futurewei.alcor.schema.Goalstateprovisioner;
 import org.springframework.stereotype.Component;
 
 import java.util.List;
+import java.util.Map;
 
 @Component
 public interface DataPlaneClient {
-    void createGoalState(GoalState goalState, String hostIp) throws Exception;
-    void createGoalState(List<HostGoalState> hostGoalStates) throws Exception;
-    void updateGoalState(List<HostGoalState> hostGoalStates) throws Exception;
-    void deleteGoalState(List<HostGoalState> hostGoalStates) throws Exception;
+    Map<String, List<Goalstateprovisioner.GoalStateOperationReply.GoalStateOperationStatus>>
```

Comment:
Could we have a brief comment here to explain each method, as well as the map (e.g., what the key is)?

Reply:
The map key is the Host IP, the value is Host ACA handled result statuses.